packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -48,7 +48,7 @@ const InputSpotlightDecorator = hoc((config, Wrapped) => {
 
 		static propTypes = /** @lends moonstone/Input/InputSpotlightDecorator.InputSpotlightDecorator.prototype */ {
 			/**
-			 * When `true`, focusing the decorator directly via 5-way will forward the focus onto the <input>
+			 * Focusing the decorator directly via 5-way will forward the focus onto the <input>.
```

Comment:
Maybe:
"Configures the decorator to focus the `<input>` when it receives focus via 5-way"
or "Focuses the `<input>` when the decorator is focused via 5-way"?

Again, suggestions welcomed.